presets.js

```diff
@@ -0,0 +1,941 @@
+exports.getPresets = function (instance) {
```

Comment:
Any of these you can accomplish using loops?

Reply:
I will reduce many lines using loops.